invenio_administration/admin.py

```diff
@@ -103,7 +103,12 @@ def add_view(self, view, view_instance, *args, **kwargs):
             view_func=view,
         )
 
-        self._menu.add_view_to_menu(view_instance)
+        from invenio_administration.views.base import AdminResourceDetailView, \
+            AdminResourceEditView
+
+        if not isinstance(view_instance, AdminResourceDetailView) and not isinstance(
```

Comment:
Personally I prefer `allow` over `block`, I think it scales better. E.g. this condition will block `DetailView` and `EditView` from being added to the menu views. But every other view will be added. For now, that means only `ListView` is added.  However, in the future we might introduce another type of view and it will be a "bug". 

It's not a major issue because we do not foresse any other types of views for now, but we never know. 

```suggestion
        if isinstance(view_instance, AdminResourceListView):
```

Reply:
what about BaseView if someone wants to go custom? this is why I went with excluding rather than picking only the list views

Reply:
That's a strong point. 

Alternatively, in case someone wants to go full custom, we could provide a `CustomView` that inherits from `BaseView` just so we can have more control on views management. That would make `BaseView` only used internally.

Reply:
Or we rename `BaseView` to just `AdminView`

Reply:
We would benefit from having a more objective views management. E.g. every view is a `BaseView`, thus we can't differentiate a `CustomView` from a `ListView` if we want to operate only on one of those. 

It is not blocking, so we can also analyse this in another issue. Especially for this `allow` or `block` situation